server/server.js

```diff
@@ -106,8 +106,18 @@ app.post('/register', (req, res) => {
         email: newUser.email
       })
         .save()
-        .then(() => {
-          res.send('Account created successfully');
+        .then(user => {
```

Comment:
Just a comment to make sure you know that this does not log in the user but just returns the user's info after they register. It's probably best to send this information back when they login.